src/NextjsLambda.ts

```diff
@@ -111,7 +113,7 @@ export class NextJsLambda extends Construct {
       timeout: functionOptions?.timeout ?? Duration.seconds(10),
       runtime: RUNTIME,
       handler: path.join(props.nextjsPath, 'server.handler'),
-      layers: [nextLayer],
+      layers: [props.nextLayer],
```

Comment:
@revmischa since the server doesn't do image optimization, is this layer even necessary?

Reply:
i remember it would error if it didn't have it, but feel free to try removing it

Reply:
Was able to remove w/o error